src/main/java/com/google/growpod/data/Plant.java

```diff
@@ -0,0 +1,88 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.growpod.data;
+
+/** Plant data class. */
+public class Plant {
+
+  /** A unique id. */
+  private String id;
+
+  /** An optional nickname. */
+  private String nickname;
+
+  private long count;
+
+  /** Foreign Key to this plant's information. */
+  private String plantTypeId;
+
+  /**
+   * Constructs a new Plant.
+   *
+   * @param id A unique ID for the plant. This value must be supplied by the user.
+   * @param nickname A nickname or null.
+   * @param count The number of this type of plant.
```

Comment:
Why not store the plant `count` with respect to a specific `adminId` in a different entity like `Inventory`?

Reply:
The `count` attribute is intended to count the number of actual plants that each `Plant` entity has.

For example, in a pea garden, you could have a `Plant: { . . . , "nickname": "West Pea Plot", "count": 42, . . . }`, which represents a plot with 42 pea plants. It would be more cumbersome to store each plant individually.

In fact, perhaps I could change `Plant` to `Plot` to make it more clear.